src/MahApps.Metro/Themes/HamburgerMenuTemplate.xaml

```diff
@@ -384,7 +384,11 @@
                                 DisplayMode="{TemplateBinding DisplayMode}"
                                 IsPaneOpen="{Binding IsPaneOpen, Mode=TwoWay, RelativeSource={RelativeSource TemplatedParent}}"
                                 IsTabStop="False"
-                                OpenPaneLength="{TemplateBinding OpenPaneLength}"
+                                OpenPaneLength="{Binding RelativeSource={RelativeSource Mode=TemplatedParent}, Path=OpenPaneLength, Mode=TwoWay}"
+                                MinimumOpenPaneLength="{TemplateBinding MinimumOpenPaneLength}"
+                                MaximumOpenPaneLength="{TemplateBinding MaximumOpenPaneLength}"
+                                CanResizeOpenPane="{TemplateBinding CanResizeOpenPane}"
+                                ResizeThumbStyle="{TemplateBinding ResizeThumbStyle}"
```

Comment:
These properties must be implemented in HamburgerMenu too.

Reply:
It should work now. The issue was that I used `TemplateBinding`. Now I use `RelativeSource TemplatedParent` instead. The Properties are in the separate file `HamburgerMenu.Properties.cs`

Reply:
👍  Can you make the `CanResizeOpenPane` default value to `false`? IMO this should be the default expectation. Thx.

Reply:
Yes, ok. I have changed this and added Options to the Demo instead.